server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/modules/blobstore/BlobStoreChoosingModule.java

```diff
@@ -75,7 +75,8 @@ BlobStoreChoosingConfiguration provideChoosingConfiguration(PropertiesProvider p
     @Singleton
     BlobStore provideBlobStore(BlobStoreChoosingConfiguration choosingConfiguration,
                                Provider<CassandraBlobStore> cassandraBlobStoreProvider,
-                               Provider<ObjectStorageBlobStore> swiftBlobStoreProvider) {
+                               Provider<ObjectStorageBlobStore> swiftBlobStoreProvider,
```

Comment:
why swift specifically ?

Reply:
Not my code, but yes, we should be s3 friendly here...